tools/autoupdate_app_sources/autoupdate_app_sources.py

```diff
@@ -106,7 +112,7 @@ def filter_and_get_latest_tag(tags, app_id):
 
 
 def tag_to_int_tuple(tag):
-    tag = tag.strip("v").strip(".")
+    tag = tag.strip("v").replace("-", ".").strip(".")
```

Comment:
about this:
it's to [support 1.13.1-2 version numbers](https://github.com/YunoHost/apps/pull/2012/commits/9647844060a7039834938435b3232dbd0e7d1d3d) because some apps are like that...

maybe we could convert `1.13.1-2` to `1.13.1.2` to compare newer versions but keep the "dash name" as normal version name in the manifest and all